scripts/install

```diff
@@ -134,59 +125,61 @@ virtualenv --python=python3 --system-site-packages "${final_path}/venv"
-
 ynh_script_progression --message="Installing app"
+
 ynh_use_nodejs
 (
 	cd "$final_path"
 	chown -R "$app": "$final_path"
 
 	# ynh_npm only exists for YunoHost 3.8+
-	"$nodejs_path/npm" install --production --unsafe-perm
+	ynh_exec_warn_less "$nodejs_path/npm" install --production --unsafe-perm
 )
 
 #=================================================
 # Setup Kresus postgresql database
 #=================================================
+ynh_script_progression --message="Creating a PostgreSQL database..."
+
+db_name=$(ynh_sanitize_dbid --db_name=$app)
+db_user=$db_name
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
 
-db_name="$app"
-db_user="$app"
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_app_setting_set --app=$app --key=db_pwd  --value=$db_pwd
 
-ynh_script_progression --message="Setting up database"
 ynh_psql_test_if_first_run
-ynh_psql_setup_db "$db_user" "$db_name"
-db_pwd=$(ynh_app_setting_get "$app" psqlpwd)
+ynh_psql_setup_db --db_user=$db_user --db_name=$db_name --db_pwd=$db_pwd
 
 #=================================================
 # CONFIGURE KRESUS
 #=================================================
 
 cp ../conf/config.ini "$final_path/config.ini"
 
-ynh_replace_string "__PORT__"          "$port"               "$final_path/config.ini"
-ynh_replace_string "__PATH__"          "$path_url"           "$final_path/config.ini"
-ynh_replace_string "__FINALPATH__"     "$final_path"         "$final_path/config.ini"
-ynh_replace_string "__SALT__"          "$salt"               "$final_path/config.ini"
-ynh_replace_string "__APP__"           "$app"                "$final_path/config.ini"
-ynh_replace_string "__DOMAIN__"        "$domain"             "$final_path/config.ini"
-ynh_replace_string "__DB_NAME__"       "$db_user"            "$final_path/config.ini"
-ynh_replace_string "__DB_USER__"       "$db_name"            "$final_path/config.ini"
-ynh_replace_string "__DB_PASSWORD__"   "$db_pwd"             "$final_path/config.ini"
-
+ynh_replace_string --match_string="__PORT__"        --replace_string="$port"       --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__PATH__"        --replace_string="$path_url"   --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__FINALPATH__"   --replace_string="$final_path" --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__SALT__"        --replace_string="$salt"       --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__APP__"         --replace_string="$app"        --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__DOMAIN__"      --replace_string="$domain"     --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__DB_NAME__"     --replace_string="$db_user"    --target_file="$final_path/config.ini"
+ynh_replace_string --match_string="__DB_USER__"     --replace_string="$db_name"    --target_file="$final_path/config.ini"
```

Comment:
Pre-existing but those two lines are inverted (`__DB_NAME__` should use `$db_user` and not `$db_name` and vice-versa)